src/bot.ts

```diff
@@ -111,12 +135,28 @@ export const robot = (app: Probot) => {
         }
 
         async function getTokenOwnerFromApi(token: string, apiUrl: string): Promise<string> {
```

Comment:
To have a better structure, here we can implement a generic function to fetch data from API:

Some example:
async function fetchFromAp(apiUrl: string, endpoint: string, query: string)
{
    const response = await axios.get(`${apiUrl}/${endpoint}/${query}?extract=owner`);

    ...
}

Reply:
Also in getTokenOwnerFromAPI we can call the above method:
...
return await fetchFromApi(apiUrl, 'tokens', token); ||
return await fetchFromApi(apiUrl, 'collections', token);

Reply:
extracted a function and used it wherever possible